java-daemon-endpoints/src/main/java/net/cryptic_game/backend/endpoints/network/NetworkMemberEndpoints.java

```diff
@@ -1,10 +1,111 @@
 package net.cryptic_game.backend.endpoints.network;
 
-import net.cryptic_game.backend.base.api.endpoint.ApiEndpointCollection;
+import com.google.gson.JsonObject;
+import net.cryptic_game.backend.base.api.endpoint.*;
+import net.cryptic_game.backend.data.device.Device;
+import net.cryptic_game.backend.data.network.Network;
+import net.cryptic_game.backend.data.network.NetworkInvitation;
+import net.cryptic_game.backend.data.network.NetworkMember;
+import net.cryptic_game.backend.data.user.User;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
 
 public class NetworkMemberEndpoints extends ApiEndpointCollection {
 
     public NetworkMemberEndpoints() {
-        super("network/member");
+        super("network_member");
+    }
+
+    @ApiEndpoint("members")
+    public ApiResponse member(@ApiParameter("user_id") final UUID userId,
+                              @ApiParameter("device") final UUID deviceId) {
+        final User user = User.getById(userId);
+        final Device device = Device.getById(deviceId);
```

Comment:
Please add a check: if device is null return an error.
Please add a check: if the user has no access to the device return an error.